qiskit/circuit/library/standard_gates/x.py

```diff
@@ -469,14 +469,6 @@ def __init__(
             "c3sx", 4, [], num_ctrl_qubits=3, label=label, ctrl_state=ctrl_state, base_gate=SXGate()
         )
 
-        if angle is not None:
-            warnings.warn(
-                "The angle argument is deprecated as of Qiskit Terra 0.17.0 and will "
-                "be removed no earlier than 3 months after the release date.",
-                DeprecationWarning,
-                stacklevel=2,
-            )
-
```

Comment:
We probably need to take out the whole keyword argument and associated handling in this class, I think?

Reply:
done in 9fa0661